README.md

```diff
@@ -166,6 +166,11 @@ This summarizes the installation process, the full installation document can be
 
 - Accelerator Prescriptive Architecture Sample [diagrams](./docs/architectures/AWS_Diagrams_Account_Network_VPC.md)
 
+### - Workshops
+
+- **The ASEA Administrator Immersion Day** workshop is focused on helping administrators who will be administering the landing zone understand how they can design, build and operate the components in ASEA. Click [here](https://catalog.us-east-1.prod.workshops.aws/v2/workshops/f3ed5d0f-d2f1-47e8-a305-168da9179aaa/en-US/sea-administrators) for an overview of the topics covered.
```

Comment:
Suggested consistency tweaks:
- ASEA has not been used or defined on this page.  Suggest changing to: "~~The ASEA~~ Accelerator Administrator Immersion Day".
- Similarly, landing zone is not used on this page (at least in this context).  Suggest changing "administering the ~~landing zone~~ understand" to "administering the **Accelerator environment to** understand".